### PR TITLE
Add curriculum overview

### PR DESCRIPTION
**Pull request by acabunoc**
I started a menu of modules we can cover as curriculum! I didn't add descriptions for all of them... I might have time this weekend

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# curricula
\ No newline at end of file
+# curricula
+
+Here are possible modules to choose from!
+
+## Understanding Open
```
Comment (RichardLitt) [inline README.md]: `Understanding Open Source`

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+* Writing a good README
+* Contributor Guidelines
+* Codes of Conduct
+* Choosing a License
+
+## Community Building
+* Personas & Pathways
+* Sustaining Momentum
+* Running Events
+* Unconcious Bias, Imposter Syndrome, Ally Skills
```
Comment (juliehache) [inline README.md]: *Unconscious

File: README.md [hunk 3]
```diff
@@ -1 +1,35 @@
-# curricula
\ No newline at end of file
+# curricula
```
Comment (juliehache) [inline README.md]: To follow the style of the document, "Curricula"

**Comment by acabunoc** [comment]
Here are a few other things that might be more suited for this audience. I don't have prepared materials or lessons for these, but there's probably expertise among us.

* Writing modular code
* Automated tests
* Continuous integration
* ... and other code-related ways to promote collaboration

**Comment by juliehache** [comment]
@acabunoc Those are really good suggestions!

File: README.md [hunk 4]
```diff
@@ -1 +1,35 @@
...
+* Unconcious Bias, Imposter Syndrome, Ally Skills
+
+## Collaboration & Leadership Developement
+* **Storytelling**: hear each others' stories to build trust and learn where people come from.
+* Skills:
+  * Active Listening
+  * Giving Feedback
+  * Asking Effective Questions
+  
+Many (not all) of these are taking from Mozilla's [Open Leadership Training Series](http://mzl.la/open-leadership).
```
Comment (kvirani) [inline README.md]: *taken

**Comment by RichardLitt** [comment]
@acabunoc If the above comments are fixed, is there anything else you need from us before merge?

**Comment by acabunoc** [comment]
Sorry I haven't had a chance to look -- I'm at MozFest right now then I'll be on vacation after. 

I completely trust you and Julie around the curriculum if you need this done soon!

**Comment by RichardLitt** [comment]
I fixed the issues, and added a coding best practices section, and merged! \o/ Thanks @acabunoc!

**Pull request merged**